app/components/DownloadApps/index.js

```diff
@@ -0,0 +1,103 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import { Link } from 'react-router-dom'
+import HdKey from 'ethereumjs-wallet/hdkey'
+import Bip39 from 'bip39'
+import CryptoJs from 'crypto-js'
+
+import Header from 'components/Header'
+import config from '../../../config'
+import { createQrImage } from 'utils/qrdisplay'
+import googlePlayBadge from 'assets/google-play-badge.png'
+import appStoreBadge from 'assets/app-store-badge.png'
+
+import { createAccount } from 'actions/account'
+
+class DownloadApps extends Component {
+
+  componentDidMount = () => {
+    createQrImage(
+      document.getElementById('qr-android'),
+      config.androidAppLink,
+      4
+    )
+
+    createQrImage(
+      document.getElementById('qr-ios'),
+      config.iOSAppLink,
+      4
+    )
+  }
+
+  handleCreateAccount = () => {
+    const mnemonic = Bip39.generateMnemonic()
+    const seed = Bip39.mnemonicToSeed(mnemonic)
+    const hdWallet = HdKey.fromMasterSeed(seed)
+    //console.log(mnemonic)
+
+    // Get the first account using the standard hd path
+    const walletHdPath = 'm/44\'/60\'/0\'/0'
+    const address = hdWallet.derivePath(walletHdPath + '/0').getWallet().getChecksumAddressString()
+
+    this.props.onCreateAccount(address, mnemonic)
+
+    this.props.history.push('/account')
+  }
+
+  handleOpenApp = (url) => (e) => {
+    chrome.tabs.create({ url })
+  }
+
+  render() {
+    return (
+      <div>
+        <Header />
```

Comment:
The pattern of using:
```
<div>
    <Header />
     ....
<div>
```

Has been used once per view, you should create a "layout" component describing it. The main idea behind it is:
* DRY
* Have a component where if you want to modify the general aspect of a view you only have to do once

Just FYI, In the ```gnosis-team-safe``` I have created the ```PageFrame```, but for you just:
```
type Props = {
  chldren: React.Node,
}
const PageFrame = ({children}: Props) => (
  <div>
    <Header />
    <main>
        { children }
    </main>
  </div>
)

export default PageFrame
```
should be enough. Let me know if you have doubts or you want to implement the "semantic" definition of views.